backend/firebaseFunctions.js

```diff
@@ -269,3 +272,62 @@ export async function checkIfUserIsInCircle(circle) {
 	}
 	return inCircle;
 }
+//NRA
+export async function makeCircleUserList(circle) {
+	console.log("start makeCircleUserList");
+	console.log("Passed circle: " + circle);
+	let circleSettingsData = [];
+	let targetUserList = Object.keys(
+		(await readData(`prayer_circle/circles/${circle}/members/`)) || {}
+	);
+	console.log("Target user list " + targetUserList);
+
+	for (targetUser in targetUserList) {
+
+		let fname = Object.values(
+			(await readData(`prayer_circle/users/${targetUser}/public/fname`)) || {}
+		);
+		let lname = Object.values(
+			(await readData(`prayer_circle/users/${targetUser}/public/lname`)) || {}
+		);
+		let name = fname + " " + lname;
+
+		let username = "";
+		const pathRef = firebase.database().ref('usernames/');
+		pathRef.orderByValue().equalTo(targetUser).once('value')
+			.then(snapshot => {
+				if (snapshot.exists()) {
+					username = snapshot.key;
+				console.log('Value found');
+				} else {
+				console.log('Value not found');
+				}
+			})
+			.catch(error => console.error(error));
```

Comment:
with the possibility of removing usernames, i think we should just access the email here. doing so should also make the code a little simpler(just use data.email that i suggested earlier)

Reply:
Should we actually just use first and last name? Showing emails would grant the circle owner access to contact all of the members, is that intentional?

Reply:
This would give every circle admin access to the circle members' contact info; do we want to do this, or should we just use their actual names instead?

Reply:
for now, leave it out. this is a conversation we can have as a team as well as with trevor

Reply:
I agree